test_runner.py

```diff
@@ -257,7 +253,7 @@ def build_test_list():
                     "--experimental.pipeline_parallel_degree 4",
                     "--experimental.pipeline_parallel_split_points layers.1,layers.2,layers.3,layers.4,layers.5,layers.6,layers.7",
                     "--experimental.pipeline_parallel_schedule interleaved_1f1b",
-                    "--model.norm_type rmsnorm",  # fused_rmsnorm throws cuda context error with pp
+                    "--model.norm_type rmsnorm",  # compiled_rmsnorm / fused_rmsnorm throws cuda context error with pp
```

Comment:
e.g. is this cuda context error the same reason as above for "crashes with PP"?

Reply:
Yes. It seems nobody understands why the error happens at this moment.

Reply:
If it is the same error, maybe we should make the commit uniform?